### DISCUSSION TITLE
Cameras stopped working

### DISCUSSION THREAD

**Discussion by ghost** [homebridge-simplisafe3/homebridge-simplisafe3#287] [Q&A] [answered]
For Homebridge SimpliSafe 3, my video doorbell cameras were working with HomeKit perfectly fine yesterday. Then I moved my homebridge server to another computer and now I can’t get the cameras to work. Any suggestions?

**Comment by shamoon** [comment, marked as answer]
Hi @drbounds , theres not really enough info to fully know what the problem is but I'd guess that when you switched servers the video software used (called ffmpeg, not part of this plugin) got messed up. Depending on which type of machine and OS youre using you may have to fix that manually or if youre using a Linux machine most of them will install it for you as part of installing this plugin. In other words you can try re-installing this.

But in general checking the Homebridge logs (and posting them here if you need help) is an important first step to figuring out most issues. You can also enable the `debug `option for more detailed output.

Hope that helps.
